### DISCUSSION TITLE
KDDFS on WSL2

### DISCUSSION THREAD

**Discussion by stsriniv** [microsoft/WSL#9102] [General]
I installed WSL2 on my Windows 11 PC.  I have a Western Digital Red NAS 3.0 hard drive that uses KDDFS.  I am trying to mount this drive using WSL2, but the drive is not even being recognized.  I tried both connecting through SATA and through a USB hard drive docking station.  If someone has successfully done this, can you please outline the steps?

**Comment by sirredbeard** [comment]
You may be interested [in this thread](https://community.wd.com/t/windows-11-built-in-linux-can-now-read-wdd-kddfs-virtual-file-system/271575/3).
